app/js/disease-associations-controller.js

```diff
@@ -107,7 +86,7 @@ angular.module('cttvControllers')
      * }
      * getFacets(filters);
      */
```

Comment:
I have to make getFacets() into a scope variable because i call it from filter-by-file-directive.js what would be a better way to handle this?